snapcraft/models/project.py

```diff
@@ -476,40 +481,40 @@ def _validate_aliases(cls, aliases):
 class Hook(models.CraftBaseModel):
     """Snapcraft project hook definition."""
 
-    command_chain: Optional[List[str]]
-    environment: Optional[Dict[str, str]]
-    plugs: Optional[UniqueStrList]
-    passthrough: Optional[Dict[str, Any]]
+    command_chain: list[str] | None = None
+    environment: dict[str, str] | None = None
+    plugs: UniqueStrList | None = None
```

Comment:
```suggestion
    plugs: UniqueList[str] | None = None
```

Reply:
Or even (since order doesn't matter here):

```suggestion
    plugs: set[str] | None = None
```